### PR TITLE
Allocating students to a project 

### PR DESCRIPTION
**Pull request by richarddushime**
Hi @rowlandm 
Here is a PR to #15 issue  
Drag and Drop enabled  and after droping automatically the database is updated 
the columns headers are the projects in Projects table 
i would need your review 
here are the screenshots 
1.
screencapture-127-0-0-1-5000-assigned-projects-2023-07-19-17_03_13

2.
screencapture-127-0-0-1-5000-assigned-projects-2023-07-19-17_04_01

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,10 +10,77 @@
...
+    conn = sqlite3.connect('student_intern_data/student_intern_data.db')
+    cursor = conn.cursor()
+    try:
+        if request.method == 'GET':
+            # Fetch the projects from the Projects table
+            cursor.execute('SELECT id, name FROM Projects ORDER BY id DESC')
+            projects = cursor.fetchall()
+
+            # Fetch the students from the Students table
+            cursor.execute('SELECT intern_id, full_name, project FROM Students')
```
Comment (rowlandm) [inline app.py:31]: As per comment - only need to get students from this intake (see Current Recruitment /current_intake)

**Comment by rowlandm** [comment]
Hi Richard,

It is very cool!

Some things that I think might need to be added that I could do later or you could do for this :

1. Add a link to the menu to access this. You could even put it under Current Recruitment (/current_intake) as a sub menu
2. The students that need to be displayed should be the ones from the Current Recruitment (/current_intake). This is because I need to work out how many students in the current intake have been assigned to each project.
3. Super minor point - on the allocation page, because the project titles are different, the student boxes don't line up vertically - it would be good to have them all start at the same vertical level.

Great work!

File: app.py (hunk 1 above)
Comment (richarddushime) [reply app.py:31]: Done

**Comment by richarddushime** [comment]
Hi 
i worked on the suggested changes 
for additionnal comments i will be waiting for your review 
Thanks 
screencapture-127-0-0-1-5000-assigned-projects-2023-07-20-08_33_29

**Comment by rowlandm** [comment]
Looks good - thanks Richard!

**Comment by richarddushime** [comment]
> Looks good - thanks Richard!

Thank you , 
you are welcome

**Pull request merged**